src/update_xs_yum.ml

```diff
@@ -320,6 +320,7 @@ let filter_file file extension =
   end else (Printf.printf "%s skipped due to wrong extension (not %s).\n%!" file extension; None)
 
 let create_tree root binpkg_iso sources_iso =
+  let open Error_to_lwt_exn in
```

Comment:
why do we still use `>>|=` in this function?

Reply:
Because some older upstream libraries that we use still return a variant type instead of throwing an exception. Hopefully in the future we'll be able to move to newer versions that throw exceptions and get rid of this module.